snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -440,7 +440,7 @@ def test_account(self, account):
                 f"'{account}' with sacctmgr: {e.stderr}"
             )
 
-        accounts = accounts.split()
+        accounts = [_.strip() for _ in accounts.split("\n") if _]
```

Comment:
Not sure why but, in my HPC, the account appears repeated 5 times. Not sure if it might affect anything, but just in case maybe we can do:
```suggestion
        accounts = set([_.strip() for _ in accounts.split("\n") if _])
```

Reply:
ugly, but yes! I will do so after a break and add a comment.

Reply:
seems to be working